src/main/java/org/kuali/kfs/coa/businessobject/lookup/KualiAccountLookupableHelperServiceImpl.java

```diff
@@ -120,29 +126,48 @@ public List<? extends BusinessObject> getSearchResults(Map<String, String> param
                  * Moved the principalByPrincipalName to a variable to more clearly see what aspect is causing the NPE
```

Comment:
Either in this user story or a follow-up user story, we should consider whether this KualiAccountLookupableHelperServiceImpl overlay can be removed. I'm not sure how much value is now added from this portion of the KFSPTS-23120 customization, now that the 06/17 patch restructured the offending code to include a null check and to retrieve the principal ID on a separate line.

Reply:
Customization removed